src/test/java/uk/gov/justice/hmpps/prison/service/receiveandtransfer/PrisonTransferServiceTest.kt

```diff
@@ -64,6 +69,12 @@ internal class PrisonTransferServiceTest {
   private val bookingLastMovementCourtWithEventId =
     getMovement(toAgencyIn = toCourt, movementReasonCode = "CRT", movementTypeCode = "CRT", eventIdIn = 123)
 
+  private val bookingLastMovementTAP =
```

Comment:
used same pattern for the other tests in the class - not very interesting and not giving a lot of value given the integration test covers the same scenarios